### PR TITLE
License updates for f formulae

### PR DESCRIPTION
**Pull request by chenrui333**
- license updates for `f` formulae
- `brew style` check

### CODE REVIEW

**Comment by chenrui333** [comment]
cancelled the workflow run.

File: Formula/freealut.rb [hunk 1]
```diff
@@ -37,7 +38,7 @@ def install
 --- a/configure.ac
 +++ b/configure.ac
 @@ -83,7 +83,8 @@ AC_DEFINE([ALUT_BUILD_LIBRARY], [1], [Define to 1 if you want to build the ALUT
- 
+
```
Comment (SeekingMeaning) [inline Formula/freealut.rb:41]: Does this break the patch?

Comment (chenrui333) [reply Formula/freealut.rb:41]: interesting, that must get slipped in with my vscode auto-format

Comment (chenrui333) [reply Formula/freealut.rb:41]: It wont break the change, but I will have this excluded for the PR. Nice catch.

Comment (SeekingMeaning) [reply Formula/freealut.rb:41]: Just wondering because I saw a few of these in a similar PR

Comment (chenrui333) [reply Formula/freealut.rb]: That might also relies on we do the patch in the `patch` block. Let me read the code too.

Comment (chenrui333) [reply Formula/freealut.rb]: DataPatch is in here, https://github.com/Homebrew/brew/blob/master/Library/Homebrew/patch.rb#L80-L101

let me quickly test the whitespace strip effect in my local.

Comment (chenrui333) [reply Formula/freealut.rb]: yeah, as expected, patching works fine (with the whitespace stripped out).
```
==> Patching
patching file configure.ac
==> ./autogen.sh
configure.ac:78: warning: The macro `AC_LIBTOOL_WIN32_DLL' is obsolete.
```

**Comment by chenrui333** [comment]
cancelled the workflow run, as not needed.

File: Formula/freealut.rb (hunk 1 above)
Comment (Rylan12) [reply Formula/freealut.rb]: I just tested it locally (although not extensively). It seems to work fine.

Comment (Rylan12) [reply Formula/freealut.rb]: Oops, I hadn't loaded the most recent comments... I concur 👍

**Review by Rylan12 — APPROVED** [review]

**Pull request merged**